README.md

```diff
@@ -20,6 +20,46 @@ ARCV Service is the service portal and API for ARCV Market.
 
 We suggest that you use the TLD `.test` as others, like `.app` may now be in the public domain and you will experience difficulty with respect to browser behavior over HTTP/HTTPS.
 
+## Versioning, branching and tags
+
+Use [semver](https://semver.org/) for versioning. Each sprint is considered a minor release until the customer asks for a major, non backwardly compatible re-write.
+
+### Sprint development
+
+ * Find the trello card and grab the URL, e.g. `https://trello.com/c/P5aKkOWJ/2056-market-roll-up-repo`
+ * Create a branch off develop that uses the ticket number and title, e.g. `dev/2056-market-roll-up-repo`
+ * Work on the ticket
+ * Raise a PR into develop
```

Comment:
Need to then pasta the PR into the Trello Card, then move the ticket to awaiting code review